### DISCUSSION TITLE
How do I make MP3s to use batteries again?

### DISCUSSION THREAD

**Discussion by miu135** [cataclysmbn/Cataclysm-BN#2040] [Q&A]
In-game or otherwise

**Comment by vorpal-void** [comment]
Open discussion about UPS in general: https://github.com/cataclysmbnteam/Cataclysm-BN/issues/1070

In current case, we have default mp3 appear as with preinstalled `"UPS conversion mod"`:

_"This device replaces conventional battery connections with a connector port that can plug into a UPS.  Any device modified with this can be used without a battery as long as you have a charged UPS with you."_

"Good", one assume and try to revert that to more convenient battery device via `magazine_battery_light_mod`:

_"A battery compartment mod that allows the use of light batteries in tools that otherwise could not."_

Activating said mod even lists mp3 (and smartphones) as valid modification target, but trying to do that will prompt message that we have no meaning of unloading our device first. 

![изображение](https://user-images.githubusercontent.com/60776130/143934635-24b9a0a6-f20c-4a0f-adff-4d6833bab031.png)
additionally, even if you manage to do that, it won't work: https://github.com/cataclysmbnteam/Cataclysm-BN/issues/1123

This is pretty strange in the world where we can disassemble and re-assemble entire wheeled houses, but can't do anything with simple electronic device.

Possible solutions: 
- allow `magazine_battery_light_mod` to be actually installed on currently default UPS-only electronics.
- obsolete or change description for `magazine_battery_light_mod` and explicitly describe what UPS actually is and how it's expected to work since almost everyone who've seen mp3 player would assume it should have interchangeable batteries, and even mp3 description itself points out that it should be using batteries:

"This _**battery-powered**_ device is loaded up with someone's music collection.  Fortunately, there's lots of songs you like, and listening to it will raise your morale slightly.  Use it to turn it on.",
"This mp3 player is turned on and playing some great tunes, raising your morale steadily while on your person.  **_It runs through batteries quickly_**; you can turn it off by using it.  It also obscures your hearing.",

- define separate mp3 player that runs on batteries and spawns independently, but I think this is not the good one since there was a reason to convert for UPS in the first place.
